deployment/HDI-Spark/Notebooks/deploy-sparkdotnet-repl.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+#
+# Install Livy and SparkMagic on Head node
+#
+
+if  [[ $HOSTNAME == hn* ]] ;
+then
+    # Update Livy Jars
+    sudo wget https://ruinliureplhdistorage.blob.core.windows.net/ruinliurepl/livy_jar.zip
+    sudo unzip livy_jar.zip
+
+    sudo cp -rf livy_jar/* /usr/hdp/current/livy2-server/
+
+    # Update SparkMagic
+    sudo wget https://ruinliureplhdistorage.blob.core.windows.net/ruinliurepl/sparkmagic.zip
+    sudo unzip sparkmagic.zip
+    
+    if [ ! -d "/usr/bin/anaconda/lib/python2.7/site-packages/sparkmagic/kernels/sparkdotnetkernel" ]
```

Comment:
Is this check necessary ?

Reply:
In case if users run the script action multiple times

Reply:
Running mkdir multiple times on an existing directory shouldn't cause an error.

Reply:
Added `-p` to resolve

Reply:
I think you can actually remove the entire `mkdir` command.   The `cp -rf` command below automatically should do this for you.

Reply:
Make sense. I have removed it. Thanks!